### PR TITLE
Using azartifacts

### PR DESCRIPTION
**Pull request by elduddz**

### CODE REVIEW

File: linux/mvn361-jdk8-cov/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-COPY ./cov-analysis-linux64-2019.12.tar.gz /tmp/cov-analysis-linux64-2019.12.tar.gz
-RUN tar -xvf "/tmp/cov-analysis-linux64-2019.12.tar.gz" -C /usr/local/bin
+    git=* \
+    && rm -rf /var/lib/apt/lists/*
+COPY --from=coverity /usr/local/bin/cov-analysis-linux64-2019.12 /usr/local/bin/cov-analysis-linux64-2019.12
 ENV COVERITY_TOOL_HOME=/usr/local/bin/cov-analysis-linux64-2019.12
-ENV PATH=${PATH}:${COVERITY_TOOL_HOME}/bin
-RUN rm -rf /tmp/*
\ No newline at end of file
+ENV PATH=${PATH}:${COVERITY_TOOL_HOME}/bin
```
Comment (eganjs) [inline linux/mvn361-jdk8-cov/Dockerfile:12]: new line at end of file

File: linux/aardvarkcov/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM debian:stretch-20190708
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+    && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+    && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+    && apt-get update && apt-get install -y --no-install-recommends \
+    git=* \
+    unzip=* \
+  && rm -rf /var/lib/apt/lists/* \
+ADD ./cov-analysis-linux64-2019.12.tar.gz /usr/local/bin
```
Comment (eganjs) [inline linux/aardvarkcov/Dockerfile:10]: new line

File: linux/node10-cov/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM aardvarkcov as coverity
+
+FROM node:10
+COPY --from=coverity /usr/local/bin/cov-analysis-linux64-2019.12 /usr/local/bin/cov-analysis-linux64-2019.12
+ENV COVERITY_TOOL_HOME=/usr/local/bin/cov-analysis-linux64-2019.12
+ENV PATH=${PATH}:${COVERITY_TOOL_HOME}/bin
```
Comment (eganjs) [inline linux/node10-cov/Dockerfile:6]: new line

File: linux/aardvarkcov/Dockerfile (hunk 2 above)
Comment (elduddz) [reply linux/aardvarkcov/Dockerfile]: done

File: linux/node10-cov/Dockerfile (hunk 3 above)
Comment (elduddz) [reply linux/node10-cov/Dockerfile]: done

**Pull request merged**